.github/renovate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    "group:allNonMajor",
+    "schedule:monthly"
+  ],
+  "reviewers": ["team:developer"],
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "groupName": "Github Action",
+      "labels": ["github_actions", "area: dependencies"],
```

Comment:
I've updated the label to be `area: github actions` to be consistent with the other `area` labels. If you update this, I'm happy to merge and see how it goes.

Reply:
Done :blush: